src/decimal128.rs

```diff
@@ -68,10 +69,25 @@ impl Decimal128 {
     /// let int = dec128.into_i32();
     /// assert_eq!(int, num);
     /// ```
+    #[deprecated(since = "0.15.0", note = "Replaced by `to_i32`")]
```

Comment:
Clippy correctly noticed that this shouldn't be named `into_i32` due to it not taking ownership of `self`. I've deprecated this and `into_u32` and added `to_{u,i}32` to replace them.